### ISSUE TITLE
Move docs to readthedocs.io

### ISSUE THREAD

**Issue by tswast** [Frozen-Flask/Frozen-Flask#80]
They support versioned docs and also can build on commit hook.

**Comment by honzajavorek** [comment]
@tswast If we're to work on this, what is your RTD username, so that I can add you as co-owner (or co-admin, forgot the exact role name atm)?

**Comment by tswast** [comment]
I'm also `tswast` on Read the Docs.

**Comment by encukou** [comment]
Any progress?

If you add me as well (also `@encukou` on RTD), I can probably find some time to set up RTD soon-ish.

*Disclaimer/full disclosure*: I'm leading a somewhat competing project ([open](https://github.com/encukou/freezeyt) but not ready). My main motivation for improving Frozen-Flask tests, documentation and determinism is for testing feature parity.

**Comment by honzajavorek** [comment]
I did not make any progress yet. I did not start the RTD page yet.

Regarding your motivation, I don't see any harm in that. Developing a competing project is a valid path for innovation. Also, let's say your project has ~5 users, but as a side effect of your work, an [existing solution used by at least ~600 projects](https://libraries.io/pypi/Frozen-Flask) gets improvements, which is great! Until the project takes off and becomes popular, majority of its impact on the humankind can actually be in form of Frozen-Flask fixes 😉
